pkg/util/downloader/commands/kubectl.go

```diff
@@ -1,18 +1,33 @@
 package commands
 
 import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"path/filepath"
+	"runtime"
+	"strings"
+
 	"github.com/loft-sh/devspace/pkg/devspace/config/constants"
 	"github.com/loft-sh/devspace/pkg/util/command"
 	"github.com/mitchellh/go-homedir"
 	"github.com/otiai10/copy"
-	"path/filepath"
-	"runtime"
-	"strings"
 )
 
 var (
-	kubectlVersion  = "v1.21.2"
-	kubectlDownload = "https://storage.googleapis.com/kubernetes-release/release/" + kubectlVersion + "/bin/" + runtime.GOOS + "/" + runtime.GOARCH + "/kubectl"
+	kubectlVersion = func() string {
+		res, err := http.Get("https://storage.googleapis.com/kubernetes-release/release/stable.txt")
+		if err != nil {
+			log.Fatal(err)
```

Comment:
we shouldn't fatal here as this will break devspace completely if this is not reachable anymore